cms/server/contest/handlers/tasksubmission.py

```diff
@@ -579,6 +579,7 @@ class UseTokenHandler(BaseHandler):
     """Called when the user try to use a token on a submission.
 
     """
+    # TODO: Decide if tokens can be applied after contest ends.
```

Comment:
The reason I mentioned this is because in #61 it was said that allowing this could make sense.

However I agree with you that letting users play tokens after the contest isn't a good idea. Usually we want the scores to be final when the contest ends.